packages/hoppscotch-cli/src/interfaces/request.ts

```diff
@@ -0,0 +1,34 @@
+import { AxiosPromise, AxiosRequestConfig } from "axios";
+import { HoppRESTRequest } from "@hoppscotch/data";
+
+/**
+ * Request stack interface.
```

Comment:
If you want describe things in documentation, make sure you are descriptive about it, what you wrote is just literally what the interface is, not what it does or any other helpful information, it is not adding anything other than noise to the code.

Reply:
Sorry for this, I will make sure to go through undocumented code.

Reply:
Not the undocumented code bit. But how useful is a comment like this:

```js
// x is a number
const x = 42;
```

Instead you should be explaining what x is doing and what is its purpose etc.

Reply:
Yes, got it, I will make the documentation more descriptive.